src/modules/main-nav/MainNav.js

```diff
@@ -11,26 +11,25 @@ export default class MainNav {
   }
 
   eventButtonMenu () {
-    const buttonMenu = this.node.querySelector('.button-closed')
-    const mainNav = this.node.querySelector('.main-nav-menu')
-    const buttonCross = this.node.querySelector('.button-closed-line')
-    console.log(mainNav)
+    const buttonMenu = this.node.querySelector('.main-nav-container__button-closed')
+    const mainNav = this.node.querySelector('.main-nav-menu__list')
+    const buttonCross = this.node.querySelector('.main-nav-container__button-closed--line')
     buttonMenu.addEventListener('click', function () {
-      mainNav.classList.toggle('main-nav-menu--opened')
-      buttonCross.classList.toggle('button-closed-line')
-      buttonCross.classList.toggle('button-closed-cross')
+      mainNav.classList.toggle('main-nav-menu__list--opened')
+      buttonCross.classList.toggle('main-nav-container__button-closed--line')
+      buttonCross.classList.toggle('main-nav-container__button-closed--line')
```

Comment:
`main-nav-container__button-closed--line` should be a constant. Look for other cases where you are repeating strings over and over.